src/libraries/System.IO.IsolatedStorage/src/System/IO/IsolatedStorage/Helper.cs

```diff
@@ -5,22 +5,22 @@ namespace System.IO.IsolatedStorage
 {
     internal static partial class Helper
     {
-        private const string IsolatedStorageDirectoryName = "IsolatedStorage";
+        private const string IsolatedStorageDirectoryName = ".isolated-storage";
```

Comment:
Does this change apply to all configurations? If so, we need to make it mobile specific as to not break everyone else.

Reply:
Updated.